lib/resources/r4/encounter.yaml

```diff
@@ -546,3 +546,92 @@ fields:
         system: http://terminology.hl7.org/CodeSystem/v3-ObservationValue
         info_link: http://hl7.org/fhir/r4/v3/ObservationValue/cs.html
     note: Encounters created with meta.security set with code 'UNCERTREL' will be returned with an external encounter type.
+
```

Comment:
As of now these will show as body fields for creates. these changes are for read and search only, correct?

Reply:
examples aren't needed either since this should only be in terminology table

Reply:
the 'required' either, i think?

Reply:
correct; updated action key, removed example and required https://github.com/cerner/fhir.cerner.com/pull/415/commits/346d1c84ec066ff89fd8f5dfdb7fec58712cc176#diff-876b1ef3d3fb228190dea594ca0a12abR550